ui/app/components/dashboard/secrets-engines-card.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+
```

Comment:
I realize we are so very not consistent with this, but could you add documentation to the js file here? You can see documentation for example in `app/components/keymgmt/distribute`. Documentation is most helpful when you're going to reuse a component and pass in params, so technically this maybe optional here, but I also find it helpful when understanding what the file is going to show.